bids/grabbids/config/bids.json

```diff
@@ -41,7 +41,7 @@
         },
         {
             "name": "run",
-            "pattern": "run-(\\d+)"
+            "pattern": "run-0*(\\d+)"
```

Comment:
I'm not sure what not capturing zeros buys you.

```Python
>>> int("06")
6
```

Reply:
It does curretly cause some values that were previously strings to become ints internally, which I think can be attributed to the way pandas auto-detects column types when reading in DataFrames. That said, you're right that it won't do anything once I introduce explicit typing to grabbit and update the BIDS spec with type declarations for all entities.